### ISSUE TITLE
Error! Binaries missing/corrupted. 

### ISSUE THREAD

**Issue by CyberBonk** [jely2002/youtube-dl-gui#546]
**Describe the bug**
gives me that error after i paste the youtube link

**To Reproduce**
1.paste this youtube link: https://www.youtube.com/watch?v=gQOv8o5lS2k&list=PLPW8O6W-1chwyTzI3BHwBLbGQoPFxPAPM&index=3

or this link (share feature): https://youtu.be/gQOv8o5lS2k?si=8rI47YqJwHF3aUVA

**Expected behavior**

**Screenshots**
![image](https://github.com/user-attachments/assets/74180b49-1d19-4eca-80e2-e07b364ab3fe)

**Additional info (please complete the following information):**
 - OS: WIN 11 23H2
 - i just downloaded it, so latest
 - Installer

**Additional context**
i disabled antivirus (windows security), and made sure that vcredist x86 is installed

searched on the wiki and in the common questions and didnt find anything related to my problem

**Comment by ArchismanKarmakar** [comment]
Same here, the 2.4.0 version is having error.
I had downloaded the 2.5.4 version, and it is fine, check that.

I am sharing the 2.5.4 version here in ZIP, the link will be valid for 90 days from 08-11-2024 22:14.

[Open-Video-Downloader-2.5.4](https://v6x3x-my.sharepoint.com/:u:/g/personal/archisman_v6x3x_onmicrosoft_com/EZR_w96FqpJHj9dEmmWKvqsBtNrdsQHgwpN085XQawDz4Q?e=Qc9K5P)

**Comment by StefanLobbenmeier** [comment]
Yeah that error is the whole reason for the fork. Please don’t reupload it as zip, share link to the releases tab instead: 
https://github.com/StefanLobbenmeier/youtube-dl-gui/releases

**Comment by ArchismanKarmakar** [comment]
@StefanLobbenmeier @ErikBrown2 @CyberBonk @itscaro @jely2002 

The version 2.5.4 is removed from the "Releases" section, that's why I shared it from my drive. Check the repository, there might be other issues too.

**Comment by StefanLobbenmeier** [comment]
Yeah it was never released from jely, since he stopped updating this repo

**Comment by ArchismanKarmakar** [comment]
@StefanLobbenmeier But why did you shift the repo here? Your account was pretty good, or you could have just made an organization from your account to resolve such issues.

**Comment by DereC4** [comment]
ty for linking the fork

**Comment by jely2002** [comment]
This issue has been fixed in v3 of the app, closing this issue as a result.
